frappe/public/js/frappe/ui/messages.js

```diff
@@ -100,7 +100,12 @@ frappe.msgprint = function(msg, title) {
 
 	if(data.message instanceof Array) {
 		data.message.forEach(function(m) {
-			frappe.msgprint(m);
+			const msg = {
```

Comment:
this was already handled in `frappe.msgprint` which is called recursively

@gshmu please check manually before you send such contributions!

Reply:
```msg = {
"message": ["message one", "message two"],
"indicator": "red",
"title": "user title"
}```
it will print "message one" and "message two", but not follow the "title" and "indicator"